### ISSUE TITLE
Input Element with Datalist not shown in MAUI Windows Desktop, Webview Runtime Issue.

### ISSUE THREAD

**Issue by carsten-riedel** [MicrosoftEdge/WebView2Feedback#3496]
**Description**
Input Element with Datalist not shown in MAUI Windows Desktop (see screenshot and sample code)

**Version**
Version Displayed: 113.0.0.0
Webview Runtime: Installed on Windows 10 V113.0.1774.42
![image](https://github.com/MicrosoftEdge/WebView2Feedback/assets/97656046/364eaf84-a67d-4aa5-a14b-e0eea94ffb8b)

Runtime: User Agent | Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.0.0 Safari/537.36 Edg/113.0.1774.42
Framework: MAUI
OS: Windows 10

**Repro Steps**
1. Create a blank .NET MAUI Blazor App Project in Visual Studio 2022
![image](https://user-images.githubusercontent.com/84492176/237190423-29d53c0d-10ac-4ab2-bafc-8615cd6d0619.png)
2. Replace Pages\Index.razor with the following
```
@page "/"
<!DOCTYPE html>
<html>
<head>
</head>
<body>
    <h1>testing</h1>

    <label for="fruit">Choose a fruit:</label>
    <input type="text" id="fruit" list="fruits">
    <datalist id="fruits">
        <option value="Apple" />
        <option value="Banana" />
        <option value="Orange" />
        <option value="Grape" />
        <option value="Mango" />
    </datalist>
</body>
</html>
```
3. Launch the App and try to select items in the Datalist. (Nothing is shown here). **Expected behavior** Datalist is shown.

![image](https://github.com/MicrosoftEdge/WebView2Feedback/assets/97656046/94ef0b52-44e0-4b54-bcdd-aba0babc3c0a)

4. That the same code inside my edge browser. Just added some debug version output.

![image](https://github.com/MicrosoftEdge/WebView2Feedback/assets/97656046/72f871c3-2a43-4132-ae85-2291d517b6a1)

## Additional context and Requirements
1. I tested all versions of MAUI changeing <MauiVersion> in csproj. -> No effect.
2. Used .NET6 and .NET 7 for additional testing -> No effect.
3. It take some time to find a workaround to adapt using a fixed version runtime, for MAUI Blazor App that's badly described. Please add some Microsoft advice for MAUI Projects. I am now using https://learn.microsoft.com/en-us/microsoft-edge/webview2/concepts/distribution#details-about-the-fixed-version-runtime-distribution-mode
See Point 4 just official advice for UWP. Please add "official advice"
4. I can now confirm using Webview Runtime 112.0.1722.68 this error does not appear and everything works fine. So the bug came in a Version > 112.0.1722.68
![image](https://github.com/MicrosoftEdge/WebView2Feedback/assets/97656046/d83c1fe2-3cd1-4557-aad0-93594d0d0154)

5. Workarounds has some pitfalls. (Let me guess MAX 260 chars for all files below the folder)
```
// fixedPath has a path limit
Environment.SetEnvironmentVariable("WEBVIEW2_BROWSER_EXECUTABLE_FOLDER", fixedPath);
```

6. Workarounds has some pitfalls.
It is not easy to add the runtime to the project. It end up to write a recursive copy task in after build and publish.  

Not working... Sorry did have the working code on this machine it's basically a recursive copy task (what's needed).
```
    <ItemGroup>
      <Content Include="Microsoft.WebView2.FixedVersionRuntime.112.0.1722.68.x64\**">
        <CopyToOutputDirectory>Always</CopyToOutputDirectory>
      </Content>
    </ItemGroup>
```
![image](https://github.com/MicrosoftEdge/WebView2Feedback/assets/97656046/99b20088-bd14-4dfc-9c79-ecbd767e438d)

Is suggest some `<WebViewRuntimePath>...<WebViewRuntimePath>`

Or a supported Nuget package.

7. There is no fixed older runtime setup .exe available to fix/test the behavior till bugs are fixed.

8. Please align with the MAUI Team closely to find a long-term solution for this dependency on WebViewRuntime, and how to use.

No effect of blazorWebView_BlazorWebViewInitializing changeing of **e** in (object sender, Microsoft.AspNetCore.Components.WebView.BlazorWebViewInitializingEventArgs e)
blazorWebView_BlazorWebViewInitializing can be used to display the e values.
```
<?xml version="1.0" encoding="utf-8" ?>
<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
             xmlns:local="clr-namespace:MauiApp6"
             x:Class="MauiApp6.MainPage"
             BackgroundColor="{DynamicResource PageBackgroundColor}">

    <!-- No effect of blazorWebView_BlazorWebViewInitializing changeing of e (object sender, Microsoft.AspNetCore.Components.WebView.BlazorWebViewInitializingEventArgs e)-->
    <!-- No effect of blazorWebView_BlazorWebViewInitializing can be used to display-->
    <BlazorWebView x:Name="blazorWebView" HostPage="wwwroot/index.html" 
                   BlazorWebViewInitializing="blazorWebView_BlazorWebViewInitializing"
                   BlazorWebViewInitialized="blazorWebView_BlazorWebViewInitialized">
        <BlazorWebView.RootComponents>
            <RootComponent Selector="#app" ComponentType="{x:Type local:Main}" />
        </BlazorWebView.RootComponents>
    </BlazorWebView>

</ContentPage>
```

**Comment by Eilon** [comment]
I created a WinUI standalone repro that shows the problem too. The `<datalist>` does NOT work on WinUI, but does work on WPF.

WinUI repro:

* App: https://github.com/Eilon/WebView2DataListBug/tree/main/WebView2DataListBug
* Repro HTML with `<datalist>` element: https://github.com/Eilon/WebView2DataListBug/blob/main/WebView2DataListBug/index.html
* Code that loads the HTML into WebView2: https://github.com/Eilon/WebView2DataListBug/blob/main/WebView2DataListBug/MainWindow.xaml.cs#L30-L41

WPF non-repro (the code is nearly identical to the WinUI app, but the bug doesn't repro in WPF):

* App: https://github.com/Eilon/WebView2DataListBug/tree/main/WpfWebView2DataListBug
* Same exact HTML as above: https://github.com/Eilon/WebView2DataListBug/blob/main/WpfWebView2DataListBug/index.html
* Code that loads the HTML into WebView2: https://github.com/Eilon/WebView2DataListBug/blob/main/WpfWebView2DataListBug/MainWindow.xaml.cs#L28-L39

**Comment by ElyssaJyu** [comment]
Thanks for provided information. We will conduct an investigation to identify the root cause, and then synchronize the findings with you.

**Comment by champnic** [comment]
Hey @Eilon & @carsten-riedel - Can you try with a Canary or Dev channel runtime? I think this is the same issue as a recent fix we put in for context menus showing up behind the app window: #3495 . If so, then the stable runtime version that began rollout today should include the fix. Let me know if that doesn't fix this issue.

**Comment by carsten-riedel** [comment]
@champnic 
I can confirm **not** working with installed runtime. 113.0.1774.50
![image](https://github.com/MicrosoftEdge/WebView2Feedback/assets/97656046/ab8c7936-1f89-401c-9459-9f29bb6c6f7c)

I can confirm using 113.0.1774.57.x64 the datalist is shown again. But hm yeah still some smaller issue, see gif video.

```
	public static MauiApp CreateMauiApp()
	{
        Environment.SetEnvironmentVariable("WEBVIEW2_BROWSER_EXECUTABLE_FOLDER", @"C:\Temp\WebViewRuntime\Microsoft.WebView2.FixedVersionRuntime.113.0.1774.57.x64");
```

![2023-05-27_07h55_57](https://github.com/MicrosoftEdge/WebView2Feedback/assets/97656046/8c42b43a-8496-4ee6-b543-2b2089a2e009)

**Comment by champnic** [comment]
@carsten-riedel Thanks for confirming the fix! I think window move blur issue might be a separate known issue: #2290

**Comment by MatthewKing** [comment]
I am seeing this same issue on 128.0.2739.79

Has there been a regression? Or was it not actually fixed?

**Comment by champnic** [comment]
@MatthewKing Can you open a new issue to track? We'll need more information (asked in the template) before we can know whether this is a regression or a new problem.

**Comment by holeinvoid** [comment]
I am seeing the same issue on  129.0.2792.89 (Official build) (64-bit). Strangely it seems the datalist does not work depending upon id attribute value. For me this does NOT work:  <input list="options" id="fabricNameAndColourName">, however this works <input list="options" id="fabricNameAndColourName2">, note prefixing id with 2. Maybe it is related to some autofill?
